src/google/cloud/happybase/table.py

```diff
@@ -232,11 +231,13 @@ def rows(self, rows, columns=None, timestamp=None,
         filters = []
         if columns is not None:
             filters.append(_columns_filter_helper(columns))
-        filters.append(_row_keys_filter_helper(rows))
+
+        row_set = _get_row_set_from_rows(rows)
```

Comment:
How confident are you that the semantics have not changed here?  The `RowFilterUnion([RowKeyRegexfilter])` bit is complicated enough that I don't know for sure.  If the semantics *have* changed, we should probably note that in this method's docstring.

Reply:
Any `Filter` filters within a row.  `RowSet` selects which rows to read, and is efficient at selecting which rows to read.  `RowKeyRegexFilter` exists and does something similar to `RowSet`, but it does it in a less efficient way.

Bottom line, `RowSet` is the more efficient alternative for the `rows` method, and keeps the semantics as they were

Reply:
OK, thanks.